libheif/plugins/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ macro(plugin_compilation name varName optionName defineName)
             message("Compiling '" ${name} "' as built-in backend")
             target_sources(heif PRIVATE ${${optionName}_sources})
             target_compile_definitions(heif PRIVATE HAVE_${defineName}=1)
-            target_include_directories(heif PRIVATE ${${varName}_INCLUDE_DIR})
+            target_include_directories(heif PRIVATE ${${varName}_INCLUDE_DIR} ${${varName}_INCLUDE_DIRS})
```

Comment:
Ditto

Reply:
I'm actually still getting compilation errors without the `${varName}_INCLUDE_DIRS` variable. Perhaps I can leave it for now and a future Pull Request can address this issue more thoroughly.

Reply:
Yes, leave it. I will take care of it later.